src/main/java/core/basesyntax/reader/FileReaderImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.reader;
```

Comment:
The package structure should be more granular to improve code organization. Consider using a `service` package for service classes like this one.